samples/EncryptionSample/EncryptionSample.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.Data.Sqlite.Core\Microsoft.Data.Sqlite.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="SQLitePCLRaw.bundle_sqlcipher" Version="1.1.8">
+      <!-- TODO: Add to Universe -->
```

Comment:
@Eilon Can we get approval to use this package in a sample?

Reply:
Approved!

Reply:
Updated. YOLO pushed https://github.com/aspnet/Universe/commit/bf23f16c227ea9ba9b4eb0dfebe2d0e9e1140647